### PR TITLE
@Rev - nitins profile

### PR DESCRIPTION
**Pull request by NITIN-KUMAR-SHARMA**
@mrmorris  -- plz review the PR and approve

### CODE REVIEW

File: nitin.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hi Nitin here
+***Enjoying Git Fundamental class ... ****
```
Comment (mrmorris) [inline nitin.md:2]: Hello! What's your fav city?

Comment (NITIN-KUMAR-SHARMA) [reply nitin.md:2]: updated

Comment (NITIN-KUMAR-SHARMA) [reply nitin.md:2]: @mrmorris  -- update the profile with fav city

**Review by NITIN-KUMAR-SHARMA — COMMENTED** [review]
updated

**Pull request merged**